screens/Content/ContentScreen.jsx

```diff
@@ -1,8 +1,9 @@
-import {useContext} from 'react';
+import React, {useContext} from 'react';
+import PropTypes from 'prop-types';
 import {TextInput, View} from 'react-native';
 import {SwiperFlatList} from 'react-native-swiper-flatlist';
 import {DNAdataContext} from '../../Data/data';
-import {renderItem} from './renderItem';
+import renderItem from './renderItem';
```

Comment:
don't use render word in the component name, try to find a better name that will explain what is inside the component

Reply:
Changed component name